erdgo/cmd/testAccount/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ElrondNetwork/elrond-sdk/erdgo"
+)
+
+func main() {
+	// Retrieving network configuration parameters
+	networkConfig, err := erdgo.GetNetworkConfig()
+	if err != nil {
+		fmt.Printf("Error getting network config: %s\n\r", err)
+		return
+	}
+
+	address := "erd18yddfyzyskajjteyj90ueackpcsp9set2ma8vp54cj4zh54hjussukc6mu"
+	// Retrieve account info from the network (balance, nonce)
+	accountInfo, err := erdgo.GetAccount(address)
+	if err != nil {
+		fmt.Printf("Error retrieving account info: %s\n\r", err)
+		return
+	}
+	balance, ok := big.NewFloat(0).SetString(accountInfo.Balance)
+	if !ok {
+		fmt.Printf("Invalid balance value: %s\n\r", err)
+		return
+	}
+	// Compute denominated balance to 18 decimals
```

Comment:
Perhaps `account.GetBalance()` to do this computation (encapsulate logic in that struct - useful for others) + associated unit tests?

Reply:
Done